### DISCUSSION TITLE
Hotwire dev newsletter

### DISCUSSION THREAD

**Discussion by joemasilotti** [therubyradar/submissions#2] [Submissions]
I think the readers of The Ruby Radar would also like to subscribe to my [Hotwire dev newsletter](https://masilotti.com/hotwire/). Edition #2 was sent out this morning, and [edition #1 is now available online](https://masilotti.com/hotwire/edition-1/) (even if you don't subscribe). I'll be publishing editions online a week or so after they are sent out.

**Comment by andrewmcodes** [comment]
Included in [Ruby Radar #8](https://rubyradar.dev/issues/ruby-radar-8-rainy-day-edition-695953)
